DependencyInjection/CmfSeoExtension.php

```diff
@@ -233,9 +233,18 @@ private function loadSitemapHandling($config, XmlFileLoader $loader, ContainerBu
     {
         $loader->load('sitemap.xml');
 
-        $container->setParameter(
-            $this->getAlias().'.sitemap.default_change_frequency',
-            $config['default_chan_frequency']
-        );
+        $container->setParameter($this->getAlias().'.sitemap.configurations', $config['configurations']);
+
+        foreach ($config['configurations'] as $key => $configuration) {
+            $container->setParameter($this->getAlias().'.sitemap.'.$key.'_configuration', $configuration);
+        }
+
+        // set a base default change frequency by now todo change that when votes respect sitemap by name
+        if (isset($config['configurations']['default']['default_change_frequency'])) {
```

Comment:
i think the best approach is to use the Configuration beforeValidation callbacks for this, not sure if it would even be possible in pure declarative code. i can have a go at that tomorrow if you want